### PR TITLE
acronym: add benchmarking

### PR DESCRIPTION
**Pull request by strangeman**

### CODE REVIEW

File: exercises/acronym/acronym_test.go [hunk 1]
```diff
@@ -12,3 +12,11 @@ func TestAcronym(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkAcronym(b *testing.B) {
+	for _, test := range stringTestCases {
+		for i := 0; i < b.N; i++ {
```
Comment (ferhatelmas) [inline exercises/acronym/acronym_test.go]: I mean this should be outer.

Comment (strangeman) [reply exercises/acronym/acronym_test.go]: Thank you for the suggestion! I fixed this. But I see this order in many other tests:
https://github.com/exercism/go/blob/master/exercises/anagram/anagram_test.go#L40
https://github.com/exercism/go/blob/master/exercises/accumulate/accumulate_test.go#L40
https://github.com/exercism/go/blob/master/exercises/bob/bob_test.go#L20

Comment (ferhatelmas) [reply exercises/acronym/acronym_test.go]: Then, we need to update them too. Do you want to send a PR for all ?

Comment (strangeman) [reply exercises/acronym/acronym_test.go]: @ferhatelmas yeah, I'll take care of them.

Comment (ferhatelmas) [reply exercises/acronym/acronym_test.go]: Thanks :smile:

**Pull request merged**